### ISSUE TITLE
.JSON functions community collection

### ISSUE THREAD

**Issue by StrangerSon71** [CraftTweaker/CraftTweaker#1395]
### Describe the feature you'd like

I think it would be helpful for there to be a place people could share their .JSON functions (Like the charcoal pit barrel example in the docs) and copy/paste others to their files.

### Describe alternatives you've considered

Everybody making their own functions.

### Additional context

I'm not entirely sure this is the right place to suggest this, if it's out of your scope feel free to just close it.

### Minecraft version

1.16

**Comment by kindlich** [comment]
Well, there was a section in CurseForge that probably noone remembers for sharing CrT scripts:
<https://www.curseforge.com/minecraft/customization/scripts/crafttweaker?filter-sort=2>

There's also this repo that contains a few example files
<https://github.com/CraftTweaker/CraftTweaker-Examples>

**Comment by StrangerSon71** [comment]
Yes, but who would want to do an external download for a function, compared to copy-pasting? And the examples, while helpful, are not the kind of thing I’m talking about.
